backend/src/v5/schemas/tickets/templates.constants.js

```diff
@@ -99,22 +101,23 @@ TemplateConstants.presetModulesProperties = {
 
 };
 
-TemplateConstants.defaultProperties = [
+TemplateConstants.defaultProperties = (config) => [
```

Comment:
No I don't mean it like this, this is too clunky 😢 

What I mean is it constructures a function like the `availableIf` callback you see in `createPropertyEntry`

Reply:
Actually I think I have a better way of refactoring this. Let me have a think and get back to this.

Reply:
@ChristosTsiotsias  I've changed this in the following way:

- `defaultProperties` is no longer exposed, no one outside of the constant file should need it. They either need to get applicable properties which calls the function, or they just want the label which should be using `basePropertyLabels`.
- This is now split into 2, `defaultProperties` - things that are not customisable and `customisableProperties` - i.e. status and pins - they take a default property object, and a function (with a config) that alters it. 


Which i believe is cleaner, if you have any thoughts, let me know. https://github.com/3drepo/3drepo.io/pull/4773/commits/0833cffeb1325186aad8601680b9297c4ead4c6c